Description: Increase acceptance test timeouts
 The pg_rewind rejoin scenario and the API switchover in pause mode use
 timeouts that are too tight on slow architectures and on loaded buildds,
 which makes the acceptance tests intermittently fail.  Raise the rejoin
 waits from 20 to 60 seconds and the switchover demote wait from 10 to 30
 seconds.
Author: Christoph Berg <myon@debian.org>
Last-Update: 2026-09-15

--- a/features/basic_replication.feature
+++ b/features/basic_replication.feature
@@ -75,9 +75,9 @@ Feature: basic replication
   Scenario: check rejoin of the former primary with pg_rewind
     Given I add the table splitbrain to postgres-0
     And I start postgres-0
-    Then postgres-0 role is the secondary after 20 seconds
+    Then postgres-0 role is the secondary after 60 seconds
     When I add the table buz to postgres-2
-    Then table buz is present on postgres-0 after 20 seconds
+    Then table buz is present on postgres-0 after 60 seconds
 
   @reject-duplicate-name
   Scenario: check graceful rejection when two nodes have the same name
--- a/features/patroni_api.feature
+++ b/features/patroni_api.feature
@@ -92,7 +92,7 @@ Scenario: check the switchover via the A
 	Then I receive a response code 200
 	And postgres-1 is a leader after 5 seconds
 	And postgres-1 role is the primary after 10 seconds
-	And postgres-0 role is the secondary after 10 seconds
+	And postgres-0 role is the secondary after 30 seconds
 	And replication works from postgres-1 to postgres-0 after 20 seconds
 	And "members/postgres-0" key in DCS has state=running after 10 seconds
 	When I issue a GET request to http://127.0.0.1:8008/primary
